src/DateTable.js

```diff
@@ -1,21 +1,22 @@
 import React from "react";
+import { Table } from "react-bootstrap";
+import { convertTimeStampToDate } from "./util";
 
-import { PersonRow } from "./PersonRow";
-
-export const DateTable = ({ participants, dates }) => {
-  const participantsArray = Object.keys(participants);
+export const DateTable = (props) => {
+  const datesArray = Object.keys(props.dates);
   return (
     <>
-      {participantsArray.map((participant, idx) => {
-        return (
-          <PersonRow
-            participant={participants[participant]}
-            participantName={participant}
-            dates={dates}
-            key={idx}
-          />
-        );
-      })}
+      <Table responsive="lg" bordered>
```

Comment:
We need to eventually add some table syntax for a11y: https://www.w3.org/WAI/tutorials/tables/one-header/